.github/workflows/python-ci.yml

```diff
@@ -4,36 +4,36 @@ on: push
 
 jobs:
   test:
-
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: ["3.11", "3.12"]
```

Comment:
In another bird, the matrix was just 3.11. What's the criterion for Python versions to test with  in these repos?

Reply:
3.12 may be required in this particular repository to support running on the newest geospatial-python docker, which supports only python 3.12 right now.